src/WebJobs.Script.WebHost/Properties/Resources.resx

```diff
@@ -125,18 +125,18 @@
   </data>
   <data name="ExecutedHttpRequest" xml:space="preserve">
     <value>Executed HTTP request: {{
-  requestId: "{mS_ActivityId}",
-  identities: "{identities}",
-  status: "{statusCode}",
-  duration: "{duration}"
+  "requestId": "{mS_ActivityId}",
```

Comment:
We have existing tests that parse this data as json and they succeed. Apparently Newtonsoft is lax and will parse correctly with missing quotes.